sdk/testing/src/test/java/io/opentelemetry/sdk/testing/assertj/MetricAssertionsTest.java

```diff
@@ -49,16 +49,18 @@
 import org.junit.jupiter.api.Test;
 
 // We always assert the contents of lists out of order to verify that assertions behave that way.
+@SuppressWarnings("Convert2MethodRef")
```

Comment:
Can you narrow the scope of this suppression to just the test function it applies to?